### ISSUE TITLE
Peer review for proposal - Group J

### ISSUE THREAD

**Issue by trnhan194** [GitHub-Traveler/COMP4010_Project2#3]
**First, this project's ambition is just impressive. My biggest impressions lie on two points:**

_**- Innovative and relevant approach:**_ Most college students like us would not think of developing an interactive 3D digital sphere model to visualize weather patterns. This concept is not only technologically intriguing, at least to me, but also highly up-to-date given the increasing impact of climate change on global weather phenomena and Vietnam's as well. (I was born and raised in Saigon, and I have never witnessed such sweltering weather in the Southern region like this year's April - real feel 48°C). 

_**Detailed plan:**_ The proposal defines tasks clearly with responsible POCs - points of contact and milestones. This will turn out to be very helpful later on when evaluating individual work at the end of the project. Besides, when having a problem, each team member can resolve it faster since he knows who the person in charge of it is. 

**On the other hand, here are a few concerns of mine:**

**_Completness of data:_** Your team mentions that there are missing data points in the datasets and states that their impacts are negligible without providing a clear justification or method for handling these gaps. Personally, to predict and visualize weather patterns effectively, accuracy should be of paramount importance. 
For this point, I would suggest your team to specify the exact percentage of missing data and propose robust methods for dealing with these gaps (e.g.: mean substitution, regression, or using a probabilistic approach like multiple imputation.)

**_Technological and technical feasibility_**: As much as I am impressed with your team's idea, I am also concerned about technical specifications of this ambitious project. 
Which technologies or tools or frameworks will you use (WebGL, Three.js, etc.)? 
Are those technologies feasible (Preliminary research or proof of concept to show feasibility)?
Besides, after choosing suitable technologies, how do you make sure each team member has the necessary skills to utilise them? Or how do you make sure all team members are confident and proficient in using those tools? (internal training or external training for team members)?

Another minor point is that since your project is ambitious and quite intensive, do you think of specifying the dates for each milestone to better track it? Or creating some "buffer" periods to cover unexpected issues (bottleneck, missing deadlines, etc.)? 

Thank you and I am looking forward to hearing your presentation. It would be my last time as a college student I guess :))

**Comment by MinhNguyen179** [comment]
Absolutely, your proposal to create an interactive 3D digital sphere model to visualize weather patterns is both creative and really timely. The usage of such technology not only demonstrates your team's technological prowess, but it also demonstrates a thorough understanding of contemporary global concerns such as climate change, which affects not only Vietnam but the entire globe.

Your comprehensive strategy, which includes clearly defined responsibilities and milestones, is commendable. This clarity will help to streamline the process, allowing for more effective troubleshooting and accountability, both of which are critical to the success of such a thorough project.

However, I share your worry about the data's completeness. The importance of data quality and completeness cannot be emphasized in a project aimed at providing accurate representations of weather patterns. Addressing missing data points with a more specific strategy, such as identifying the proportion of missing data and using strong statistical imputation methods (mean substitution, regression techniques, or multiple imputation), will improve your project's dependability.

Another important consideration is technological feasibility. It's amazing to strive for the usage of modern technologies like WebGL or Three.js, but the practical implications of such technology must be carefully considered (I believe that Three.js might be a great choice in my opinion). Keeping all team members up to date through internal or external training sessions is critical for sustaining progress and reaching project milestones effectively.

Furthermore, given the ambitious nature of the project, incorporating a more detailed timetable with exact dates for each milestone, as well as buffer periods for unanticipated delays, would be a wise strategy to managing the project timeline successfully.

**Comment by tienvu95** [comment]
Thanks @trnhan194 and @MinhNguyen179 for the detailed feedback and response. I don't have anything to add, given this level of details
